opgincidentresponse/actions/incident_notifications.py

```diff
@@ -1,3 +1,14 @@
 from response.core.models import Incident
 from response.slack.models import CommsChannel
 from response.slack.decorators import recurring_notification, single_notification
+
+
+@single_notification()
+def take_a_break(incident: Incident):
```

Comment:
Not a big deal, but the name should probably match:
```suggestion
def incident_response_process(incident: Incident):
```

Reply:
Ha! that teaches me not to copy and paste 😄